lib/test/contracts/testWrapper.ts

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+import { ExtendTruffleContract } from "../../ExtendTruffleContract";
+import { Utils } from "../../utils";
+const SolidityContract = Utils.requireContract("AbsoluteVote");
+import ContractWrapperFactory from "../../ContractWrapperFactory";
+
+export class TestWrapperWrapper extends ExtendTruffleContract {
+
+    constructor() {
```

Comment:
this file is using tabs or 4 spaces where every other one uses 2 spaces

Reply:
Turns out there is a known bug in tslint.  The rule that enforces line indentation doesn't work, hasn't for a while now.

https://github.com/palantir/tslint/issues/2814